docker/check-status.sh

```diff
@@ -1,8 +1,14 @@
-. /tmp/iip-configuration.sh
+#!/bin/bash
 
-COUNT=$(ps aux | grep /opt/iipsrv/src/iipsrv.fcgi | grep -v grep | wc -l)
-if [ $COUNT -lt $NB_IIP_PROCESS ]; then
-    echo "$COUNT / $NB_IIP_PROCESS on $(date)"
-    killall -9 /opt/iipsrv/src/iipsrv.fcgi
-    /tmp/start-iip.sh
-fi;
+while true; do
+    COUNT=$(ps aux | grep /tmp/start-iip-proc.sh | grep -v grep | wc -l)
+        if [ $COUNT -lt $NB_IIP_PROCESS ]; then
```

Comment:
We'll now trust that the iipsrv fastcgi processes will respawn automatically based on our changes in start-iip-proc. That being said - if the start-iip-proc processes crash for some reason, nothing will handle that respawn - so switching this back to the original mechanism for checking-status, but we check the start-iip-proc status rather than iipsrv.fcgi status.

Reply:
I'm not understanding why we need any of the check-status.sh anymore now.. can we just remove it, and the references to it in docker/deploy.sh and docker/Dockerfile?

Reply:
no you need to have it so that in the event that any of the start-iip-proc.sh processes stop for whatever reason, this status check can get us back into a state where we have NB_IIP_PROCESS start-iip-proc.sh and NB_IIP_PROCSS iipsrv.fcgi processes

Reply:
when/why would those processes stop though? they're just a simple forever bash loop essentially, right? I thought the main thing that was "stopping"/dying was the iipsrv.fcgi processes, which seem to be fully handled by start-iip.sh/start-iip-proc.sh now.

Reply:
No idea why - but if they did, then those iipsrv processes would not respawn. If you don't have one of those forever-bash-loops running per-port, then iipsrv.fcgi can't respawn - so if we want to assume that those start-iip-proc processes will never fail, then we can get rid of this.

Reply:
🤷‍♂️ works fine I guess, and hopefully we'll never need it, but in case we do it will at least eventually self-heal by this with the side-effects you noted but again, probably not worth lingering on those side-effects like you mentioned since it's unlikely/theoretical to happen in the first place.